meta.yaml

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
This file needs to be in `recipes/flags/meta.yaml` not in the top-level of the repo.

Reply:
It's important to put this file in the correct directory or the CI will not be able to even attempt to build it.